tasks.py

```diff
@@ -0,0 +1,18 @@
+from invoke import task
+from os import path, getcwd
+
+@task
+def extracredit(
+    c,
+    division,
+    results,
+    survey=path.join('credit', 'survey.csv'),
+    config=path.join('..', 'config', 'development.py'),
```

Comment:
I think we might want to change the script to use the production config directly since it's only going to be used on production servers anyway. I got the extra-credit results this way.

```suggestion
    config=path.join('..', 'config', 'production.py'),
```

Reply:
I actually think trying to run locally via SSH tunnel like I suggested in the docs is a better way than to try exec'ing into the container, in which case using the development config is preferable. The user can always pass an alternate configuration which has their own preferences, or temporarily make edits as needed. 

Furthermore, using the production config is a slight hindrance as people who do chose to run locally won't have a copy of the production conf on hand.

Reply:
Also after further consideration I think you have a point, so I'm going to accept the change.